src/components/contactUsForm/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+
+import {
+  FormContainer,
+  StyledForm,
+  StyledInput,
+  StyledTextArea,
+  FormButton
+} from "./Styles";
+
+const ContactUsForm = () => {
+  const initialValues = { name: "", phone: "", email: "", message: "" };
+
+  const [values, setValues] = useState(initialValues);
+
+  const handleChange = e => {
+    const { name, value } = e.target;
+    setValues({ ...values, [name]: value });
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    sendEmail();
+    setValues(initialValues);
+  };
+
+  const sendEmail = () => {
+    const url =
+      "https://pe6m7vr3qk.execute-api.us-west-2.amazonaws.com/default/emilio-sendgrid-lamda";
+
+    fetch(url, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(values)
+    })
+      .then(response => console.log("Email was sent."))
```

Comment:
you probably will want to clean up this log output before utilizing this to make it more professional.